src/components/Statistics.tsx

```diff
@@ -104,7 +104,7 @@ export default function Statistics({ setShowStats }: Props) {
   const options = { year: "numeric", month: "short", day: "numeric" };
   const event = new Date();
   // @ts-ignore
-  const unambiguousDate = event.toLocaleDateString("en-CA", options);
+  const unambiguousDate = event.toLocaleDateString(locale, options);
```

Comment:
When sharing result with others, I'm assuming we want to format according to current locale. However, if we share to the entire planet, let's say, on Twitter, some users might prefer to use English.